.env.example

```diff
@@ -0,0 +1,3 @@
+// This should be change when we are in production
+DOMAIN_API_GATEWAY=localhost
+PORT_API_GATEWAY=3001
```

Comment:
This should be only a URL, like `API_URL=https://localhost:3000`

Reply:
No, I've try in that way, but axios don't recognize in that way. I have to configure it like this
```
axios.create({
  baseURL: domain,
  proxy: {
     port
  }
})
```
If I put port in BaseURL axios don't recognize it and throw an error

Reply:
What if you try with `baseURL: 'http://localhost:3000/` directly without using env vars to see if works